### DISCUSSION TITLE
一个配合vscode的使用问题

### DISCUSSION THREAD

**Discussion by kmcs42** [tuna/thuthesis#606] [Q&A]
最近在使用该模板配合vscode的latex插件写论文，发现每处有引用文献（使用\cite{}命令）的地方都会出现警告（黄色下划线提示）：citation ‘xxxx’ on page X undefined.
(xxxx是文献名，X是页码)
但是不影响编译，想问下有没有解决办法或者能如何关掉提示……满屏幕警告还挺烦人的。

**Comment by xueruini** [comment]
discussion 跟 issue 有啥区别？

**Reply by Harry-Chen** [reply]
discussion 主要用来问一些使用的问题，issue 还是倾向于建议相关的吧
